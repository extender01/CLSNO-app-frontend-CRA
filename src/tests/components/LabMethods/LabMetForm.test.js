import React from 'react';
import {shallow} from 'enzyme';
import LabMetForm   from '../../../components/LabMethods/LabMetForm';
import {labmets} from '../../fixtures/labmets';

test('should render LabMetForm for adding method with empty inputs', () => {
    const wrapper = shallow(<LabMetForm />);
    expect(wrapper).toMatchSnapshot();
});

test('should render LabMetForm for editing method with filled inputs', () => {
    const particularTest = labmets[1];
    const wrapper = shallow(<LabMetForm labMetToEdit={particularTest} />);
    expect(wrapper).toMatchSnapshot();
});

test('should render error when name or where is missing on form submission', () => {
    const wrapper = shallow(<LabMetForm />);
    expect(wrapper).toMatchSnapshot();

    //finds first form tag - at(0) - on rendered LabMetForm and simulates submit function
    //2nd argument is object which is normally used as argument, here we fake it to avoid..
    //..error saying things are undefined, here preventDefault as empty function
    wrapper.find('form').at(0).simulate('submit', {
        preventDefault: () => {}
    });

    expect(wrapper.state('error').length).toBeGreaterThan(0);
    expect(wrapper).toMatchSnapshot();
});

test('should set name state on input change', () => {
    const newValue = 'nova hodnota';
    const wrapper = shallow(<LabMetForm />);
    wrapper.find('input').at(0).simulate('change', {
        target: {
            name: 'name',
            value: newValue
        }
    });
    expect(wrapper.state('name')).toBe(newValue);

    
});

// const inputCheck = (inputNumber, inputName) => {
//     const newValue = 'nova hodnota';
//     const wrapper = shallow(<LabMetForm />);
//     wrapper.find('input').at(inputNumber).simulate('change', {
//         target: {
//             name: inputName,
//             value: newValue
//         }
//     });
//     expect(wrapper.state(inputName)).toBe(newValue);
// };

const inputCheck = (inputNumber, inputName) => {
    test(`should check input ${inputName}`, () => {

        const newValue = 'nova hodnota';
        const wrapper = shallow(<LabMetForm />);
        wrapper.find('input').at(inputNumber).simulate('change', {
            target: {
                name: inputName,
                value: newValue
            }
        });
        
        
        expect(wrapper.state(inputName)).toBe(newValue);
        
    });
   
};

inputCheck(0, 'name');

