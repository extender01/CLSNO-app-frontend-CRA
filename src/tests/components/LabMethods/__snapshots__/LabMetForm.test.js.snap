// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LabMetForm for adding method with empty inputs 1`] = `
<div>
  <form
    className="labmetform"
    onSubmit={[Function]}
  >
    <ul>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Název testu
        </label>
        <input
          name="name"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Synonyma
        </label>
        <input
          name="syn"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Kde se dělá
        </label>
        <input
          list="where"
          name="where"
          onChange={[Function]}
          value=""
        />
        <datalist
          id="where"
        >
          <option
            value="SNO"
          />
          <option
            value="Krnov"
          />
        </datalist>
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Název v Akordu
        </label>
        <input
          name="nameAk"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Skupiny v akordu
        </label>
        <input
          name="groupAk"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Odběr
        </label>
        <input
          name="draw"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Preanalytika
        </label>
        <input
          name="preanal"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Poznámka
        </label>
        <input
          name="note"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Metodika
        </label>
        <input
          name="metodics"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Jednotka
        </label>
        <input
          name="unit"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Odbornost
        </label>
        <input
          name="expertise"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
    </ul>
    <ul>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Čas rutina
        </label>
        <input
          name="rutTime"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Statim
        </label>
        <input
          name="statCare"
          onChange={[Function]}
          type="checkbox"
          value={false}
        />
        <label>
          Pohotovost
        </label>
        <input
          name="erCare"
          onChange={[Function]}
          type="checkbox"
          value={false}
        />
      </li>
      <li
        className="f_between labmetform__li"
      />
      <li
        className="f_between labmetform__li"
      >
        <label>
          Dohláška
        </label>
        <input
          name="additionalOrder"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Poznámka k dostupnosti
        </label>
        <input
          name="responseNote"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Snižující interference
        </label>
        <input
          name="interfereDown"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Zvyšující interference
        </label>
        <input
          name="interfereUp"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Fyziologická variabilita
        </label>
        <input
          name="physVar"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Biologický poločas
        </label>
        <input
          name="bioHalfLife"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
    </ul>
    <ul>
      <li
        className="f_center labmetform__li"
      >
        <button>
          Uložit
        </button>
      </li>
    </ul>
  </form>
  <div
    className="f_stretch"
  >
    <form
      className="labmetform__ref-form g1"
      onSubmit={[Function]}
    >
      <ul>
        <li
          className="f_center labmetform__li"
        >
          <label>
            Referenční meze:
          </label>
        </li>
        <li
          className="f_between labmetform__li"
        >
          <label>
            Pohlaví:
          </label>
          <select
            name="sex"
            onChange={[Function]}
            value="U"
          >
            <option
              value="U"
            >
              Uni
            </option>
            <option
              value="M"
            >
              Muž
            </option>
            <option
              value="F"
            >
              Žena
            </option>
          </select>
        </li>
        <li
          className="f_between labmetform__li"
        >
          <label>
            Věk:
          </label>
          <input
            name="age"
            onChange={[Function]}
            placeholder="Např. 0 - 15 let"
            type="text"
            value=""
          />
        </li>
        <li
          className="f_between labmetform__li"
        >
          <label>
            Ref. mez:
          </label>
          <input
            name="range"
            onChange={[Function]}
            placeholder="Např. 0,25 - 0,60"
            type="text"
            value=""
          />
        </li>
        <li
          className="f_between labmetform__li"
        >
          <label>
            Poznámka k rozmezí:
          </label>
          <input
            name="refNote"
            onChange={[Function]}
            placeholder="Poznámka k mezi"
            type="text"
          />
        </li>
        <li
          className="f_center labmetform__li"
        >
          <button>
            Přidej ref. meze
          </button>
        </li>
      </ul>
    </form>
  </div>
</div>
`;

exports[`should render LabMetForm for editing method with filled inputs 1`] = `
<div>
  <form
    className="labmetform"
    onSubmit={[Function]}
  >
    <ul>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Název testu
        </label>
        <input
          name="name"
          onChange={[Function]}
          type="text"
          value="CYFRA"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Synonyma
        </label>
        <input
          name="syn"
          onChange={[Function]}
          type="text"
          value="cyp je to"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Kde se dělá
        </label>
        <input
          list="where"
          name="where"
          onChange={[Function]}
          value="Krnov"
        />
        <datalist
          id="where"
        >
          <option
            value="SNO"
          />
          <option
            value="Krnov"
          />
        </datalist>
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Název v Akordu
        </label>
        <input
          name="nameAk"
          onChange={[Function]}
          type="text"
          value="S_EXT_CYFRA"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Skupiny v akordu
        </label>
        <input
          name="groupAk"
          onChange={[Function]}
          type="text"
          value="EXT_VSE"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Odběr
        </label>
        <input
          name="draw"
          onChange={[Function]}
          type="text"
          value="sg"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Preanalytika
        </label>
        <input
          name="preanal"
          onChange={[Function]}
          type="text"
          value="standard"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Poznámka
        </label>
        <input
          name="note"
          onChange={[Function]}
          type="text"
          value="baf"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Metodika
        </label>
        <input
          name="metodics"
          onChange={[Function]}
          type="text"
          value="CLIA"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Jednotka
        </label>
        <input
          name="unit"
          onChange={[Function]}
          type="text"
          value="UI/ml"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Odbornost
        </label>
        <input
          name="expertise"
          onChange={[Function]}
          type="text"
          value="801"
        />
      </li>
    </ul>
    <ul>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Kdy odjíždí externí
        </label>
        <input
          name="whenExtTransport"
          onChange={[Function]}
          type="text"
          value="po-pa"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Jak často externí
        </label>
        <input
          name="extHowOften"
          onChange={[Function]}
          type="text"
          value="denne"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Odezva externí
        </label>
        <input
          name="extResponse"
          onChange={[Function]}
          type="text"
          value="3 dny"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Odezva externí
        </label>
        <input
          name="extResponse"
          onChange={[Function]}
          type="text"
          value="3 dny"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Kdo posílá z lab
        </label>
        <input
          name="parcelWho"
          onChange={[Function]}
          type="text"
          value="RIA"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Lab preanalytika
        </label>
        <input
          name="parcelPreanal"
          onChange={[Function]}
          type="text"
          value="stocit stahnout"
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Lab poznámka
        </label>
        <input
          name="parcelNote"
          onChange={[Function]}
          type="text"
          value="nesrat se s tim"
        />
      </li>
    </ul>
    <ul>
      <li
        className="f_center labmetform__li"
      >
        <button>
          Uložit
        </button>
      </li>
    </ul>
  </form>
  <div
    className="f_stretch"
  />
</div>
`;

exports[`should render error when name or where is missing on form submission 1`] = `
<div>
  <form
    className="labmetform"
    onSubmit={[Function]}
  >
    <ul>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Název testu
        </label>
        <input
          name="name"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Synonyma
        </label>
        <input
          name="syn"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Kde se dělá
        </label>
        <input
          list="where"
          name="where"
          onChange={[Function]}
          value=""
        />
        <datalist
          id="where"
        >
          <option
            value="SNO"
          />
          <option
            value="Krnov"
          />
        </datalist>
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Název v Akordu
        </label>
        <input
          name="nameAk"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Skupiny v akordu
        </label>
        <input
          name="groupAk"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Odběr
        </label>
        <input
          name="draw"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Preanalytika
        </label>
        <input
          name="preanal"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Poznámka
        </label>
        <input
          name="note"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Metodika
        </label>
        <input
          name="metodics"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Jednotka
        </label>
        <input
          name="unit"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Odbornost
        </label>
        <input
          name="expertise"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
    </ul>
    <ul>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Čas rutina
        </label>
        <input
          name="rutTime"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Statim
        </label>
        <input
          name="statCare"
          onChange={[Function]}
          type="checkbox"
          value={false}
        />
        <label>
          Pohotovost
        </label>
        <input
          name="erCare"
          onChange={[Function]}
          type="checkbox"
          value={false}
        />
      </li>
      <li
        className="f_between labmetform__li"
      />
      <li
        className="f_between labmetform__li"
      >
        <label>
          Dohláška
        </label>
        <input
          name="additionalOrder"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Poznámka k dostupnosti
        </label>
        <input
          name="responseNote"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Snižující interference
        </label>
        <input
          name="interfereDown"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Zvyšující interference
        </label>
        <input
          name="interfereUp"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Fyziologická variabilita
        </label>
        <input
          name="physVar"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Biologický poločas
        </label>
        <input
          name="bioHalfLife"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
    </ul>
    <ul>
      <li
        className="f_center labmetform__li"
      >
        <button>
          Uložit
        </button>
      </li>
    </ul>
  </form>
  <div
    className="f_stretch"
  >
    <form
      className="labmetform__ref-form g1"
      onSubmit={[Function]}
    >
      <ul>
        <li
          className="f_center labmetform__li"
        >
          <label>
            Referenční meze:
          </label>
        </li>
        <li
          className="f_between labmetform__li"
        >
          <label>
            Pohlaví:
          </label>
          <select
            name="sex"
            onChange={[Function]}
            value="U"
          >
            <option
              value="U"
            >
              Uni
            </option>
            <option
              value="M"
            >
              Muž
            </option>
            <option
              value="F"
            >
              Žena
            </option>
          </select>
        </li>
        <li
          className="f_between labmetform__li"
        >
          <label>
            Věk:
          </label>
          <input
            name="age"
            onChange={[Function]}
            placeholder="Např. 0 - 15 let"
            type="text"
            value=""
          />
        </li>
        <li
          className="f_between labmetform__li"
        >
          <label>
            Ref. mez:
          </label>
          <input
            name="range"
            onChange={[Function]}
            placeholder="Např. 0,25 - 0,60"
            type="text"
            value=""
          />
        </li>
        <li
          className="f_between labmetform__li"
        >
          <label>
            Poznámka k rozmezí:
          </label>
          <input
            name="refNote"
            onChange={[Function]}
            placeholder="Poznámka k mezi"
            type="text"
          />
        </li>
        <li
          className="f_center labmetform__li"
        >
          <button>
            Přidej ref. meze
          </button>
        </li>
      </ul>
    </form>
  </div>
</div>
`;

exports[`should render error when name or where is missing on form submission 2`] = `
<div>
  <h2>
    Vyplň povinné položky Název a kde se to dělá
  </h2>
  <form
    className="labmetform"
    onSubmit={[Function]}
  >
    <ul>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Název testu
        </label>
        <input
          name="name"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Synonyma
        </label>
        <input
          name="syn"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Kde se dělá
        </label>
        <input
          list="where"
          name="where"
          onChange={[Function]}
          value=""
        />
        <datalist
          id="where"
        >
          <option
            value="SNO"
          />
          <option
            value="Krnov"
          />
        </datalist>
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Název v Akordu
        </label>
        <input
          name="nameAk"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Skupiny v akordu
        </label>
        <input
          name="groupAk"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Odběr
        </label>
        <input
          name="draw"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Preanalytika
        </label>
        <input
          name="preanal"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Poznámka
        </label>
        <input
          name="note"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Metodika
        </label>
        <input
          name="metodics"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Jednotka
        </label>
        <input
          name="unit"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Odbornost
        </label>
        <input
          name="expertise"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
    </ul>
    <ul>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Čas rutina
        </label>
        <input
          name="rutTime"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Statim
        </label>
        <input
          name="statCare"
          onChange={[Function]}
          type="checkbox"
          value={false}
        />
        <label>
          Pohotovost
        </label>
        <input
          name="erCare"
          onChange={[Function]}
          type="checkbox"
          value={false}
        />
      </li>
      <li
        className="f_between labmetform__li"
      />
      <li
        className="f_between labmetform__li"
      >
        <label>
          Dohláška
        </label>
        <input
          name="additionalOrder"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Poznámka k dostupnosti
        </label>
        <input
          name="responseNote"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Snižující interference
        </label>
        <input
          name="interfereDown"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Zvyšující interference
        </label>
        <input
          name="interfereUp"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Fyziologická variabilita
        </label>
        <input
          name="physVar"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
      <li
        className="f_between labmetform__li"
      >
        <label>
          Biologický poločas
        </label>
        <input
          name="bioHalfLife"
          onChange={[Function]}
          type="text"
          value=""
        />
      </li>
    </ul>
    <ul>
      <li
        className="f_center labmetform__li"
      >
        <button>
          Uložit
        </button>
      </li>
    </ul>
  </form>
  <div
    className="f_stretch"
  >
    <form
      className="labmetform__ref-form g1"
      onSubmit={[Function]}
    >
      <ul>
        <li
          className="f_center labmetform__li"
        >
          <label>
            Referenční meze:
          </label>
        </li>
        <li
          className="f_between labmetform__li"
        >
          <label>
            Pohlaví:
          </label>
          <select
            name="sex"
            onChange={[Function]}
            value="U"
          >
            <option
              value="U"
            >
              Uni
            </option>
            <option
              value="M"
            >
              Muž
            </option>
            <option
              value="F"
            >
              Žena
            </option>
          </select>
        </li>
        <li
          className="f_between labmetform__li"
        >
          <label>
            Věk:
          </label>
          <input
            name="age"
            onChange={[Function]}
            placeholder="Např. 0 - 15 let"
            type="text"
            value=""
          />
        </li>
        <li
          className="f_between labmetform__li"
        >
          <label>
            Ref. mez:
          </label>
          <input
            name="range"
            onChange={[Function]}
            placeholder="Např. 0,25 - 0,60"
            type="text"
            value=""
          />
        </li>
        <li
          className="f_between labmetform__li"
        >
          <label>
            Poznámka k rozmezí:
          </label>
          <input
            name="refNote"
            onChange={[Function]}
            placeholder="Poznámka k mezi"
            type="text"
          />
        </li>
        <li
          className="f_center labmetform__li"
        >
          <button>
            Přidej ref. meze
          </button>
        </li>
      </ul>
    </form>
  </div>
</div>
`;
